ament_cmake_pytest/cmake/ament_add_pytest_test.cmake

```diff
@@ -89,6 +91,11 @@ function(ament_add_pytest_test testname path)
     "--junit-prefix=${PROJECT_NAME}"
   )
 
+  if(DEFINED ARG_WERROR AND ARG_WERROR)
```

Comment:
I don't think you need `DEFINED ARG_WERROR` here.

Reply:
Addressed in 73d7a6c